build.rs

```diff
@@ -33,17 +33,32 @@ fn main() {
     println!("cargo:rerun-if-changed=migration/migration");
     println!("cargo:rerun-if-changed=migration/idempotent");
     println!("cargo:rerun-if-changed=templates");
+    println!("cargo:rerun-if-changed=migration/migration-table.sql");
+    println!("cargo:rerun-if-changed=bootstrap.sql");
 
-    let mut sql = include_str!("migration/migration-table.sql").to_string();
     let rendered_sql = render_sql().expect("unable to render migrations");
-    sql.push_str(rendered_sql.as_str());
 
     let manifest_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
-    let mut out_path = PathBuf::from(manifest_dir);
+    let mut out_path = PathBuf::from(manifest_dir.clone());
     out_path.push(MIGRATION_FILE_NAME);
     let mut file = File::create(out_path).unwrap();
-    file.write_all(sql.as_bytes())
+    file.write_all(rendered_sql.as_bytes())
         .expect("unable to write file contents");
+
+    let mut bootstrap_path = PathBuf::from(manifest_dir);
+    bootstrap_path.push("bootstrap.sql");
+    let mut migration_sql = include_str!("migration/migration-table.sql").to_string();
```

Comment:
should we move these files into
- migration/bootstrap/migration-table.sql
- migration/bootstrap/schema.sql
?

Reply:
done